src/components/TodoList/TodoList.tsx

```diff
@@ -1 +1,37 @@
-export const TodoList = () => {};
+import { TodoInfo } from '../TodoInfo';
+
+interface Todo {
+  id: number,
+  title: string,
+  completed: boolean,
+  userId: number,
+}
+
+type Props = {
+  todos: Todo[],
+};
+
+export const TodoList: React.FC<Props> = ({ todos }) => {
+  return (
+    <>
```

Comment:
I know I said you about fragment (haven't seen all code), but it's better to use semantic tags here (`ul`, `li` inside each TodoInfo)